README.md

```diff
@@ -1,2 +1,71 @@
 # commercetools-email-retry-processor
-Scheduled service which allows to send emails asynchronously and retry on errors.
+
+ The email retry processor fetches all emailobjects, which are saved in customobjects,  of the given tenants. For each
+ of this emailobjects, the processor checks, if they are in state "pending" and ,if true, it triggers a api endpoint.
+ This api endpoint should contain the email delivery logic.
```

Comment:
`The email retry processor fetches all email objects, which are saved as Custom Objects in CTP,  of the given tenants. For each email object in "pending" state, this job triggers an api endpoint. This triggered api endpoint should contain the email delivery logic.`

Reply:
1. (For each email object in "pending" state) what are other states the email objects should have?

Reply:
1634447

The other state are described below